docker-compose.yaml

```diff
@@ -44,21 +49,51 @@ services:
     image: ghcr.io/darpa-askem/data-service:latest
     environment:
       DATASET_STORAGE_BASE_URL: s3://datasets/
+      STORAGE_HOST: http://minio:9000
+      S3_BUCKET: askem
+      S3_DATASET_PATH: datasets
+      S3_RESULT_PATH: simulations
```

Comment:
Will now also need `S3_ARTIFACT_PATH: artifacts`

Reply:
Don't worry about the docker-compose

Reply:
Probably true, but not worrying about it for this commit. Can add that later once this is done and we are doing something with artifacts via docker compose.